src/Entity/Lector.php

```diff
@@ -2,22 +2,33 @@
 
 namespace App\Entity;
 
+use Doctrine\Common\Collections\ArrayCollection;
 use Doctrine\ORM\Mapping as ORM;
 
 /**
  * @ORM\Entity(repositoryClass="App\Repository\LectorRepository")
  */
-class Lector
+class Lector extends User
 {
     /**
-     * @ORM\Id()
-     * @ORM\GeneratedValue()
-     * @ORM\Column(type="integer")
+     * @ORM\ManyToMany(targetEntity="Tag")
+     * @ORM\JoinTable(name="lector_specialization",
```

Comment:
`lector_specialization` na
`lector_specializations`

je to many to many

Reply:
`lectors_specializations` spíš